### PR TITLE
The vxid passed to the ip command must be decimal.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: shakenfist/net.py [hunk 1]
```diff
@@ -209,9 +209,12 @@ def update_floating_gateway(self, gateway):
 
     def subst_dict(self):
         # NOTE(mikal): it should be noted that the maximum interface name length
-        # on Linux is 15 user visible characters.
+        # on Linux is 15 user visible characters, we therefore use hex for vxids
+        # where they appear in an interface nate. Note that vx_id does not appear
```
Comment (mandoonandy) [inline shakenfist/net.py:213]: nate?

**Review by mandoonandy — CHANGES_REQUESTED** [review]
The spelling nazi rides again.

File: shakenfist/net.py (hunk 1 above)
Comment (mikalstill) [reply shakenfist/net.py:213]: Its short for Nathan I believe.

Comment (mandoonandy) [reply shakenfist/net.py:213]: Maaate

**Comment by mikalstill** [comment]
Self merge after verbal approval from Andy v2.

**Pull request merged**